commons/_shared/bs-lib-components/ui/src/components/layout/BsLayoutDrawer.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+<QDrawer
+    v-bind="drawerProps"
+    :mini-width="collapsedWidth"
+    :width="expandedWidth"
```

Comment:
There are three reasons:
- First one is that `QDrawer` shifts `QHeader` and `QPage` to the right, so it doesn't overlap when you have tabs.
- Second one is that the `BsDrawerBtn.vue` component is attached to the `QDrawer`, moving it back and forth doesn’t require some weird css modifications.
- Third one is that the animation is more responsive and the user sees the drawer emerge from the side as soon as he clicks the button, and if it was toggled, there would have been a delay because emerging from the tabs would take som of that time (and because of that button would behave a bit weird too).